### ISSUE TITLE
Issues in command help text

### ISSUE THREAD

**Issue by andy-maier** [pywbem/pywbemtools#355]
* `class delete`:
  - Grammar issue in "Use this with command with caution.". 
  - The following sentence from the docs would also make sense in the help text: "Many WBEM servers may not allow this operation or may severely limit the conditions under which a class can be deleted from the server."
* `class associators/references`:
  - The description claims that subclasses are also returned, which is not the case.

No issue with #1 and #2 above

**COMMENT:KS** RE: subclasses on class associators/references. Is that documented anywhere???. I just tested and pegasus does return just the direct classes apparently 

wbemcli -n ophttp class associators  CIM_Profile --no
//leonard/root/cimv2:CIM_AccessControlInformation
//leonard/root/cimv2:CIM_AuthenticationRequirement
//leonard/root/cimv2:CIM_AuthorizedPrivilege
//leonard/root/cimv2:CIM_BaseMetricDefinition
//leonard/root/cimv2:CIM_BaseMetricValue
//leonard/root/cimv2:CIM_Capabilities
//leonard/root/cimv2:CIM_Collection
//leonard/root/cimv2:CIM_Credential
//leonard/root/cimv2:CIM_DiagnosticService
//leonard/root/cimv2:CIM_Identity
//leonard/root/cimv2:CIM_Job
//leonard/root/cimv2:CIM_ListenerDestination
//leonard/root/cimv2:CIM_Location
//leonard/root/cimv2:CIM_ManagedElement
//leonard/root/cimv2:CIM_MethodParameters
//leonard/root/cimv2:CIM_OrganizationalEntity
//leonard/root/cimv2:CIM_PolicyRoleCollection
//leonard/root/cimv2:CIM_PolicySet
//leonard/root/cimv2:CIM_PowerManagementService
//leonard/root/cimv2:CIM_Product
//leonard/root/cimv2:CIM_Profile
//leonard/root/cimv2:CIM_RecordForLog
//leonard/root/cimv2:CIM_RedundancySet
//leonard/root/cimv2:CIM_RegisteredProfile
//leonard/root/cimv2:CIM_Role
//leonard/root/cimv2:CIM_ScopedSettingData
//leonard/root/cimv2:CIM_SecuritySensitivity
//leonard/root/cimv2:CIM_Service
//leonard/root/cimv2:CIM_ServiceAccessPoint
//leonard/root/cimv2:CIM_SettingData
//leonard/root/cimv2:CIM_SoftwareIdentity
//leonard/root/cimv2:CIM_StatisticalData
//leonard/root/cimv2:CIM_StatisticalInformation
//leonard/root/cimv2:CIM_System
//leonard/root/cimv2:CIM_UsersAccess
//leonard/root/cimv2:CIM_View

Note that CIM_Job has subclasses

wbemcli -n ophttp class tree CIM_Job
CIM_Job
 +-- CIM_PrintJob
 +-- CIM_ConcreteJob

but they are not in the first list so one assumes that the result is just the direct associations.

**Comment by andy-maier** [comment]
I checked the description of Associators in DSP0200 1.4, and it does not have an explicit statement about subclasses for the class level use. However, the description: "The Associators operation enumerates CIM objects (classes or instances) associated with a particular source CIM object." can be interpreted such that only the associated instances or classes are to be returned. For instance level use, it cannot be any other way technically, because it is strictly at the instance level without specifying any class. For class level use, it could technically include subclasses of the associated classes.

Then I checked DSP0223 1.1.0, and it says "The GetAssociatedClassesWithPath operation traverses an association from a class on a starting end to classes on all of its far ends and returns the associated CIM classes together with their class paths."
That is pretty much the same, also without being explicit on subclasses.

Bottom line, I am pretty sure that not including subclasses on class level association is correct, and I think the standards text supports that sufficiently, although not perfectly. Time for another Mantis issue to queue?

**Comment by andy-maier** [comment]
Fixed in PR #366.
